modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -4,41 +4,50 @@
 [id="op-installing-pipelines-operator-in-web-console_{context}"]
 = Installing the {pipelines-title} Operator in web console
 
-You can install {pipelines-title} using the Operator listed in the {product-title} OperatorHub. When you install the {pipelines-title} Operator, the Custom Resources (CRs) required for the Pipelines configuration are automatically installed along with the Operator.
+You can install {pipelines-title} using the Operator listed in the {product-title} OperatorHub. When you install the {pipelines-title} Operator, the custom resources (CRs) required for the pipelines configuration are automatically installed along with the Operator.
+
+The default Operator custom resource definition (CRD) `config.operator.tekton.dev` is now replaced by `tektonconfigs.operator.tekton.dev`.  In addition, the operator provides the following additional CRDs to individually manage OpenShift Pipelines components:
+ `tektonpipelines.operator.tekton.dev`, `tektontriggers.operator.tekton.dev` and `tektonaddons.operator.tekton.dev`.
+
+If you have OpenShift Pipelines already installed on your cluster, the existing installation is seamlessly upgraded. The operator will replace the instance of `config.operator.tekton.dev` on your cluster with an instance of `tektonconfigs.operator.tekton.dev` and additional objects of the other CRDs as necessary.
+
+[WARNING]
+====
+If you manually changed your existing installation, such as, changing the target namespace in the `config.operator.tekton.dev` CRD instance by making changes to the `resource name - cluster` field, then the upgrade path is not smooth. In such cases, the recommended work flow is to uninstall your installation and reinstall the {pipelines-title} Operator.
+====
+
+The {pipelines-title} Operator now provides the option to choose the components that you want to install by specifying profiles as part of the `TektonConfig` CR. The `TektonConfig` CR is automatically installed when the Operator is installed.
+The supported profiles are:
+
+* Basic: This installs only Tekton pipelines.
+* Default: This installs Tekton pipeline and Tekton triggers.
+* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components: Tekton Pipelines, Tekton Triggers, Tekton Addons(which include ClusterTasks, ClusterTriggerBindings, ConsoleCLIDownload, ConsoleQuickStart and ConsoleYAMLSample).
```

Comment:
Addons, which include ` ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStar`t and `ConsoleYAMLSample`.